diff --git a/node_modules/react-native-timezone/android/src/main/java/com/samitha/timezone/TimezoneModule.kt b/node_modules/react-native-timezone/android/src/main/java/com/samitha/timezone/TimezoneModule.kt
index e0fe509..6cb923a 100644
--- a/node_modules/react-native-timezone/android/src/main/java/com/samitha/timezone/TimezoneModule.kt
+++ b/node_modules/react-native-timezone/android/src/main/java/com/samitha/timezone/TimezoneModule.kt
@@ -1,5 +1,6 @@
 package com.samitha.timezone
 
+import android.content.Context
 import android.content.ContentResolver
 import android.os.Build
 import android.provider.Settings
